test/internal/outgoing/dispatcher_test.go

```diff
@@ -80,10 +80,8 @@ func TestDispatchEvents_Error(t *testing.T) {
 		TargetId:   "product_id_2",
 	}, store, throttler, documentStores)
 
-	time.Sleep(50 * time.Millisecond)
```

Comment:
we changed to use channel and wait for destinations to be handled in previous PR so it's safe to remove